tests/integration/message/message.spec.ts

```diff
@@ -87,10 +86,11 @@ describe('Message Integration Tests - Happy Path', () => {
       const response = await requestSender.getMessageById({ pathParams: { id } });
 
       expect(response).toSatisfyApiSpec();
-      expect(response.status).not.toBe(httpStatusCodes.OK);
+      expect(response.status).toBe(httpStatusCodes.OK);
```

Comment:
Where do you check that the results match the one you requested?
steps in this test:
add new with id
get - result matches the one you added